### PR TITLE
Add Arch installation and fix flags order

### PR DESCRIPTION
**Pull request by Paiusco**
- [x] I have added a news fragment under `changelog.d/` (if the patch affects the end users)

## Summary of changes

- Add Arch as Linux option on installation steps

### CODE REVIEW

**Comment by chrysle** [comment]
Thanks! The order has already been corrected in #1338, which will be merged soon.

**Comment by Paiusco** [comment]
@chrysle my bad, I'll remove the parameter fix, but will leave only the Arch addition then.

File: README.md [hunk 1]
```diff
@@ -61,6 +61,14 @@ pipx ensurepath
 sudo pipx ensurepath --global # optional to allow pipx actions with --global argument
 ```
 
+- Arch:
+
+```
+sudo pacman -S python-pipx
+pipx ensurepath
+sudo pipx --global ensurepath # optional to allow pipx actions with --global argument
```
Comment (dukecat0) [inline README.md:69]: I just opened #1397 for fixing it and `sudo pipx ensurepath --global` should work now.
```suggestion
sudo pipx ensurepath --global # optional to allow pipx actions with --global argument
```

**Comment by dukecat0** [comment]
Please add a changelog entry as well.

**Comment by Paiusco** [comment]
Should it be added to the changelog even tho it doesn't affect the end users? I don't see a reason to flood the changelog just for adding a new instruction for a distro

**Review by gaborbernat — APPROVED** [review]

**Comment by huxuan** [comment]
Hi @Paiusco, the pull request need to be updated for merge.

**Pull request merged**